source/index.html.md

```diff
@@ -528,7 +528,7 @@ POST https://gateway.clearhaus.com/authorizations
 ````
 
 Authorizations can be created using different payment methods:
-`card`, `applepay`, `mobilepayonline`.
+`card`, `applepay`, `googlepay`, `mobilepayonline`.
```

Comment:
The ordering here aligns with the ordering below, but any reason for squeezing `googlepay` in (signaling some kind of ordering I don't understand) rather than adding it to the end?

Reply:
Just phonetically I liked the two `*pay` words to follow each other :)